modules/check_in/app/workers/check_in/travel_claim_submission_worker.rb

```diff
@@ -28,20 +28,25 @@ def perform(uuid, appointment_date)
 
       logger.info("Submitting travel claim for #{uuid}, #{appointment_date}")
 
-      claims_resp = TravelClaim::Service.build(
-        check_in: check_in_session,
-        params: { appointment_date: }
-      ).submit_claim
-
-      claim_number = claims_resp.dig(:data, :claimNumber)&.last(4)
+      claims_resp =
+        begin
+          TravelClaim::Service.build(
+            check_in: check_in_session,
+            params: { appointment_date: }
+          ).submit_claim
+        rescue
+          logger.error("Error calling BTSSS Service for: #{uuid}, #{appointment_date}")
+          nil
```

Comment:
If you `return nil` rather than setting claims_resp to nil, that would solve future issues of handling nil values in claims_resp, wouldn't it? It prevents the rest of the code from running. Or if you want the rest of the code to run anyway, setup another function perhaps to handle the nil value path vs the good path. Some ideas if you wanted to short-circuit the `claims_resp` as `nil`

Reply:
Woops, this is Lee, sorry I commented with the wrong account